src/button/buttonview.js

```diff
@@ -64,64 +146,62 @@ export default class ButtonView extends View {
 					// We can't make the button disabled using the disabled attribute, because it won't be focusable.
 					// Though, shouldn't this condition be moved to the button controller?
 					if ( this.isEnabled ) {
-						this.fire( 'click' );
+						this.fire( 'execute' );
```

Comment:
Because you can do the same with keyboard Return/Space keys, touch it on your smartphone, etc. `click` events are limited to the mouse and developers who use `ButtonView` don't actually need to know what kind of interaction executed the button. `execute` is an interface which is uniform and brings no confusion.